src/components/DefaultImage.js

```diff
@@ -0,0 +1,60 @@
+// @flow
+
+import React from 'react';
+
+type DefaultImageProps = {
+  /**
+   * The address of the image you want to display.
+   */
+  imageSrc: string;
+  /**
+   * The address of a default image to display if there's an issue displaying the prefered image address.
+   */
```

Comment:
Properties which have a default value shouldn't be set to have an optional ("?") type. They always, in fact, have a value because of the fact that they are defined in the XXXDefaultProps type and are initialized in the defaultProps object. (Also, the type of a property in the XXXDefaultProps type should always be identical to its type in the XXXProps type.)